components/email/src/Email.svelte

```diff
@@ -863,6 +864,58 @@
       ? MessageType.DRAFTS
       : MessageType.MESSAGES;
   }
+
+  interface DisplayedParticipant extends Participant {
+    _type: "to" | "cc" | "bcc";
```

Comment:
`_type` to avoid conflict with API response payloads. **_** to indicate that it's local to this component.